### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by Rockyraven**
some refactor of css and HTML

### CODE REVIEW

File: Quickstore/pages-styles/landingpage.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import url('/Quickstore/pages-styles/color.css');
+@import url('/Quickstore/pages-styles/footer.css');
 *{
     margin: 0px;
```
Comment (bharati-21) [inline Quickstore/pages-styles/landingpage.css:4]: You could add these resets for all pages in a common CSS sheet.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged.

**Review by rishmi5h — APPROVED** [review]
looks good to merge

**Comment by Rockyraven** [comment]
thanku for review

**Pull request merged**